sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/RxDocumentClientImpl.java

```diff
@@ -628,7 +628,17 @@ public void init(CosmosClientMetadataCachesSnapshot metadataCachesSnapshot, Func
                     this.clientTelemetryConfig,
                     this,
                     this.connectionPolicy.getPreferredRegions());
-            clientTelemetry.init();
+            clientTelemetry.init().doOnTerminate(() -> {
+                logger.info(
+                    "Initialized DocumentClient [{}] with machineId[{}]"
+                        + " serviceEndpoint [{}], connectionPolicy [{}], consistencyLevel [{}], directModeProtocol [{}]",
+                    clientId,
+                    ClientTelemetry.getMachineId(diagnosticsClientConfig),
+                    serviceEndpoint,
+                    connectionPolicy,
+                    consistencyLevel,
+                    configs.getProtocol());
```

Comment:
What benefit does the protocol provides? If we are trying to get connectionMode, then it will come as part of connectionPolicy logging. This is redundant in my opinion. 
Also protocol is always TCP, because Cosmos DB no longer supports Direct mode with HTTP protocol.

Reply:
I have just used the same log as we today use when initializing a client - we have a log line "Initializing ..." - to allow correlation I kept the same logging details in the "Initialized ..." (as in initialization completed)